app/models/profile.rb

```diff
@@ -13,6 +13,6 @@ def ensure_image_from_github
   private def fetch_profile_image_from_github
     client = Octokit::Client.new
     url = client.user(user.authentication_provider_github.uid.to_i).avatar_url
-    URI.open(url)
+    URI.open(url) # standard:disable Security/Open
```

Comment:
To suppress warnings, change as follows

```ruby
URI.parse(url).open
```

However, doing so results in the following error with `steep check`.

```
app/models/profile.rb:16:19: [error] Type `(::URI::File | ::URI::FTP | ::URI::HTTP | ::URI::HTTPS | ::URI::LDAP | ::URI::LDAPS | ::URI::MailTo | ::URI::WS | ::URI::WSS | ::URI::Generic)` does not have method `open`
│ Diagnostic ID: Ruby::NoMethod
│
└     URI.parse(url).open
                     ~~~~
```